src/WingetCreateCore/Models/Partials/InstallerPartials.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.WingetCreateCore.Models.Installer
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using YamlDotNet.Serialization;
+
+    /// <summary>
+    /// Partial class that implements cloning functionality to the PackageDependencies class.
+    /// </summary>
+    public partial class PackageDependencies : ICloneable
+    {
+        /// <summary>
+        /// Creates a new PackageDependencies object that is a copy of the current instance.
+        /// </summary>
+        /// <returns>A new PackageDependencies object that is a copy of the current instance.</returns>
+        public object Clone()
+        {
+            return new PackageDependencies { MinimumVersion = this.MinimumVersion, PackageIdentifier = this.PackageIdentifier };
+        }
+    }
+
+    /// <summary>
+    /// Partial Installer class for defining a string type ReleaseDateTime field.
+    /// Workaround for issue with model generating ReleaseDate with DateTimeOffset type.
+    /// </summary>
+    public partial class Installer
+    {
+        /// <summary>
+        /// Gets or sets the Release Date time.
+        /// </summary>
+        [YamlMember(Alias = "ReleaseDate")]
+        [RegularExpression(@"^(0[1-9]|1[012])[- /.](0[1-9]|[12][0-9]|3[01])[- /.](19|20)\d\d$")]

```

Comment:
>"^(0[1-9]|1[012])[- /.](0[1-9]|[12][0-9]|3[01])[- /.](19|20)\d\d$") [](http://example.com/codeflow?start=28&length=67)

update regex

Reply:
Added customValidation for dateTime field